packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx

```diff
@@ -533,3 +533,19 @@ export const getSubnetsString = (data: Subnet[]) => {
     ? firstThreeSubnetsString.concat(`, plus ${data.length - 3} more.`)
     : firstThreeSubnetsString;
 };
+
+export const getVPCIPv4 = (
+  interfaceWithVPC: Interface | LinodeInterface | undefined
+) => {
+  if (interfaceWithVPC) {
+    if ('purpose' in interfaceWithVPC) {
+      return interfaceWithVPC.ipv4?.vpc;
+    } else {
+      return interfaceWithVPC.vpc?.ipv4?.addresses.find(
+        (address) => address.primary
+      )?.address;
+    }
```

Comment:
We don't need the `else` since we're returning
```suggestion
    }
    return interfaceWithVPC.vpc?.ipv4?.addresses
      .find((address) => address.primary)?.address;
```